src/pages/help-centre/deriv-x.js

```diff
@@ -168,7 +203,14 @@ const DerivXRealMoneyAccount = () => (
         <Text>
             <Localize
                 translate_text="To deposit funds into your Deriv X account on Deriv, you’ll need to use the funds in your Deriv account. Go to Cashier > <0>Transfer between accounts</0> and follow the instructions on the screen."
-                components={[<LocalizedLinkText color="red" key={0} />]}
+                components={[
+                    <ExternalLink
+                        href={`${deriv_x_app_url}/account-transfer`}
```

Comment:
This should be pointing to `https://app.deriv.com/cashier/account-transfer`. You can use the `deriv_app_url` for this

Reply:
okay thanks